Library/Formula/vultr.rb

```diff
@@ -0,0 +1,38 @@
+require "language/go"
+
+class Vultr < Formula
+  desc "Command-line tool for Vultr"
+  homepage "https://jamesclonk.github.io/vultr"
+  url "https://github.com/JamesClonk/vultr/archive/v1.5.tar.gz"
+  sha256 "ca373d2748268b822e4ad5aeeb4ee8150f8c55c2d761e6c2c8913657469dcca5"
+  head "https://github.com/JamesClonk/vultr.git"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  go_resource "github.com/kr/fs" do
+    url "https://github.com/kr/fs.git", :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
+  end
+
+  go_resource "golang.org/x/tools" do
+    url "https://github.com/golang/tools.git", :revision => "473fd854f8276c0b22f17fb458aa8f1a0e2cf5f5"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git", :revision => "8b27f58b78dbd60e9a26b60b0d908ea642974b6d"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/JamesClonk/"
+    ln_sf buildpath, buildpath/"src/github.com/JamesClonk/vultr"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "godep", "go", "build", "-o", "vultr", "."
+    bin.install "vultr"
+  end
+
+  test do
+    system bin/"vultr", "version"
```

Comment:
@mikemcquaid Not really unfortunately.  :disappointed: 
`version` is the only command which does not need a Vultr API key to be used.
All API calls require a key (even for ready-only public information).